superset/sql/parse.py

```diff
@@ -550,7 +550,7 @@ def _parse(cls, script: str, engine: str) -> list[exp.Expression]:
             last_statement = statements.pop()
             target = statements[-1]
             for node in statements[-1].walk():
-                if hasattr(node, "comments"):
+                if hasattr(node, "comments"):  # pragma: no cover
```

Comment:
There were a couple lines that I couldn't hit with tests, but I thought it was safer to keep them, so I added the pragma.